cli/internal/k8s/common.go

```diff
@@ -34,13 +34,18 @@ func init() {
 }
 
 func getRestConfig() *rest.Config {
-	homePath, err := os.UserHomeDir()
-	if err != nil {
-		message.Fatal(nil, "Unable to load the current user's home directory")
+	// use the KUBECONFIG context if it exists
+	configPath := os.Getenv("KUBECONFIG")
+	if configPath == "" {
+		// use the current context in the default kubeconfig in the home path of the user
+		homePath, err := os.UserHomeDir()
+		if err != nil {
+			message.Fatal(nil, "Unable to load the current user's home directory")
```

Comment:
What's the latest on doing this vs returning `error`? I thought at some point I had heard @jeff-mccoy say we were switching to returning `error` to make testing easier.

Though, I'm seeing right below this it doing the same thing, so maybe the question is whether a refactor of this method is warranted along with this change.

Reply:
we are moving towards bubbling errors, but that would be a larger refactor than this code right now, so I think this one is okay.